packages/twenty-front/src/modules/object-record/record-table/record-table-cell/components/RecordTableCellHotkeysEffect.tsx

```diff
@@ -0,0 +1,118 @@
+import { useContext } from 'react';
+import { Key } from 'ts-key-enum';
+
+import { FieldContext } from '@/object-record/record-field/contexts/FieldContext';
+import { useClearField } from '@/object-record/record-field/hooks/useClearField';
+import { useIsFieldClearable } from '@/object-record/record-field/hooks/useIsFieldClearable';
+import { useIsFieldInputOnly } from '@/object-record/record-field/hooks/useIsFieldInputOnly';
+import { useToggleEditOnlyInput } from '@/object-record/record-field/hooks/useToggleEditOnlyInput';
+import { RecordIndexHotkeyScope } from '@/object-record/record-index/types/RecordIndexHotkeyScope';
```

Comment:
Isn't time of have a barrel file that exports all of these?

```tsx
export * from './object-record/record-field/hooks`
```

```tsx
import { useClearField, useIsFieldClearable, useIsFieldInputOnly, useToggleEditOnlyInput } from '@/object-record/record-field/hooks';
```

Could be even better and expose `record-field` as a module by putting a barrel file higher up and expose anything from contexts and hooks.